python/examples/colors.py

```diff
@@ -0,0 +1,84 @@
+"""
+Atom property coloring in chemiscope
+====================================
+
+This example demonstrates how to color atoms based on scalar
+properties.
+
+Note that the same parameters can be used with `chemiscope.show`
+to visualize an interactive widget in a Jupyter notebook.
+"""
+
+import ase.io
+import chemiscope
+import numpy as np
+
+# loads a dataset of structures
+frames = ase.io.read("data/alpha-mu.xyz", ":")
+
+# converts the arrays from the format they are stored in to an array
+# format that can be processed by the ASE utilities
+for a in frames:
+    a.arrays["polarizability"] = np.array(
+        [
+            (axx + ayy + azz) / 3
+            for (axx, ayy, azz) in zip(
+                a.arrays["axx"], a.arrays["ayy"], a.arrays["azz"]
+            )
+        ]
+    )
+
+    # one possible measure of anisotropy...
+    a.arrays["alpha_eigenvalues"] = np.array(
+        [
+            np.linalg.eigvalsh([[axx, axy, axz], [axy, ayy, ayz], [axz, ayz, azz]])
+            for (axx, ayy, azz, axy, axz, ayz) in zip(
+                a.arrays["axx"],
+                a.arrays["ayy"],
+                a.arrays["azz"],
+                a.arrays["axy"],
+                a.arrays["axz"],
+                a.arrays["ayz"],
+            )
+        ]
+    )
+
+    a.arrays["anisotropy"] = (
+        a.arrays["alpha_eigenvalues"][:, 2] - a.arrays["alpha_eigenvalues"][:, 0]
+    )
```

Comment:
Do we really need this in the "atomic coloring" example?

Reply:
I don't know what you mean by that.

Reply:
well, I thought it was better to avoid adding new data files, but we need scalar properties to display. 
I'll add a comment so it's clear it's not like one needs to do this in order to use the coloring.